Set single primary font (Patron) (#1306)
redpanda-data/documentation: modules/get-started/pages/config-rpk-profile.adoc

Reviewer: Not part of the diff, but if possible would you mind fixing the xref on L22 to `#specify-configuration-properties`?

All `rpk` commands can read configuration values from a profile. You pass a profile to an `rpk` command by setting the `--profile` flag. For example, the command `rpk topic produce dev-topic --profile dev` gets its configuration from the profile named `dev`.

> == Quick start

Reviewer: ```suggestion
== Quickstart
```
For consistency across the docs

Create a profile with authentication and TLS to quickly set up cluster access instead of using environment variables or connection flags:

[,bash]
----
rpk profile create <profile-name> \
  --set brokers=<hostname:port> \
  --set admin.hosts=<hostname:port> \
  --set user=<username> \
  --set pass=<password> \
  --set sasl.mechanism=<mechanism> \
  --set tls.enabled=true \
  --description "<profile-description>"
----

Replace `<mechanism>` with your desired SASL mechanism (`SCRAM-SHA-256`, `SCRAM-SHA-512`, or `PLAIN`).

> When you create a profile, rpk automatically switches to use that profile so you don't have to keep passing `--profile` flags every time.

Reviewer: ```suggestion
When you create a profile, rpk automatically switches to use that profile so you don't need to pass `--profile` flags every time.
```

> Check what profile you're using with:

Reviewer: ```suggestion
Check the active profile:
```
For conciseness

[,bash]
----
rpk profile current
----

> Now all commands use this profile automatically:

Reviewer: ```suggestion
Now all `rpk` commands use this profile automatically:
```
Would you typically run into an error or expect to see something returned in the terminal if you happen to not be using the correct profile?

Reviewer 2: I think this is somewhat well understood. If you're not using the correct profile, you would use different credentials, etc. It's like a google chrome profile.

All `rpk` commands can read configuration values from a profile. You pass a profile to an `rpk` command by setting the `--profile` flag. For example, the command `rpk topic produce dev-topic --profile dev` gets its configuration from the profile named `dev`.

== Quick start

Create a profile with authentication and TLS to quickly set up cluster access instead of using environment variables or connection flags:

[,bash]
----
rpk profile create <profile-name> \
  --set brokers=<hostname:port> \
  --set admin.hosts=<hostname:port> \
  --set user=<username> \
  --set pass=<password> \
  --set sasl.mechanism=<mechanism> \
  --set tls.enabled=true \
  --description "<profile-description>"
----

Replace `<mechanism>` with your desired SASL mechanism (`SCRAM-SHA-256`, `SCRAM-SHA-512`, or `PLAIN`).

Use your profile for all commands:

[,bash]
----
rpk topic list --profile <profile-name>
rpk topic create <topic-name> --profile <profile-name>

> ----

Reviewer 3: It's worth noting that when you create a profile rpk automatically swaps to use that profile.

Reviewer 2: changed the flow to better convey this info
